package.json

```diff
@@ -85,9 +85,12 @@
 		"express-validator": "^6.15.0",
 		"helmet": "^6.0.1",
 		"jsonwebtoken": "^9.0.0",
+		"mailgun": "^0.5.0",
 		"mailgun-js": "^0.22.0",
+		"mailgun.js": "^8.2.1",
```

Comment:
Just wondering - what's the reason to use two different packages for mailgun?

Reply:
I have deleted the extra one, thank you for pointing it out.